test/jdk/sun/security/ssl/SSLSessionImpl/MultiNSTParallel.java

```diff
@@ -73,7 +73,7 @@ static class ClientThread extends Thread {
         public void run() {
             String name = Thread.currentThread().getName();
             SSLSession r;
-            System.err.println("waiting " + Thread.currentThread().getName());
+            System.err.println(name + " is ready");
```

Comment:
This test is printing some things to System.out and some to System.err. It's also using TLSBase which is printing everything to System.err. This will make the logs difficult to read and understand.

Reply:
With the threading though, some of the errors need to be printed right away.  I understand your concern, but my attempt is to make stdout display obvious results without searching all the debugging info.  In some cases catching an exception is a valid result, which may look like a test failure if they were all in one output buffer.